python/ml4ir/applications/classification/pipeline.py

```diff
@@ -168,6 +170,31 @@ def get_relevance_dataset(
 
         return relevance_dataset
 
+    def finish(self, job_status, job_info):
+        # This will write the _SUCCESS/_FAILURE files
+        super().finish(job_status, job_info)
+
+        # Create model_bundle.zip depending on user args
```

Comment:
@balikasg Should this even be in ml4ir? Probably worth doing it internally? Also I'm interested in understanding why you need to save vocabulary files? I thought you were using the `VocabLookup` in the model itself?

CC: @jakemannix

Reply:
@lastmansleeping , we need the labels at this point. The model predicts a list of ids, we want to be able to convert to the actual label and post process.

Reply:
We are planning to work on a post-processing function tf.function, that will create the format needed e.g., a list of ordered predictions with their probabilities. But we are not there yet.

Reply:
Why not do it in ml4ir itself?

Reply:
https://github.com/salesforce/ml4ir/blob/master/python/ml4ir/base/model/serving.py#L128

@balikasg @jakemannix